docs/network-spec/connection-manager.tex

```diff
@@ -0,0 +1,1641 @@
+% TODO:
```

Comment:
It seems that in this commit the `connection-manager.tex` file is added but not included by `network-spec.tex`.

Reply:
Yes, I was debugging something and forgot to include it 😅